src/test/java/org/jenkinsci/plugins/p4/DefaultEnvironment.java

```diff
@@ -156,7 +158,7 @@ private ManualWorkspaceImpl createWorkspace(String path) {
 		String stream = null;
 		String line = "LOCAL";
 		String view = "\"" + path + "\"" + " //" + client + "/" + filename;
-		WorkspaceSpec spec = new WorkspaceSpec(true, true, false, false, false, false, stream, line, view, null, null, null, true);
+		WorkspaceSpec spec = new WorkspaceSpec(true, true, false, false, false, false, stream, null,line, view, null, null, null, true);
```

Comment:
You wouldn't need to change the tests if you added a Deprecated method signature to match the old pipeline api.